Grids: Describe renderAsync (#3001)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDataGrid/1 Configuration/scrolling/mode.md

A user scrolls a single page only. Scrolling is available only if [all the page's rows](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging/pageSize.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/#pageSize') do not fit into the UI component's height. In this mode, the [pager](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/pager '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/pager/') performs the main navigation and scrolling is auxiliary. If [paging](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/paging '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/') is disabled, the UI component loads all rows simultaneously which can reduce the UI component's performance. In this case, we recommend that you use another scrolling mode.

- **Virtual**       
Rows are loaded when they get into the viewport and removed once they leave it. In this mode, users can move to any page instantly.

>     If the rows take time to be loaded and rendered, they display gray boxes. [Rendering optimization](/concepts/05%20UI%20Components/DataGrid/10%20Enhance%20Performance%20on%20Large%20Datasets/030%20Rendering%20Optimization.md '/Documentation/Guide/UI_Components/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization') can reduce rendering time and remove the gray boxes. Alternatively, you can disable the **scrolling**.[renderAsync]({basewidgetpath}/Configuration/scrolling/#renderAsync) property, but please be notified that this may lower the performance.

Reviewer: ```suggestion
    If the rows take time to be loaded and rendered, they display gray boxes. [Rendering optimization](/concepts/05%20UI%20Components/DataGrid/10%20Enhance%20Performance%20on%20Large%20Datasets/030%20Rendering%20Optimization.md '/Documentation/Guide/UI_Components/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization') can reduce rendering time and remove the gray boxes. Alternatively, you can disable the **scrolling**.[renderAsync]({basewidgetpath}/Configuration/scrolling/#renderAsync) property, but this can impact performance.
```
